tests/test_urldispatch.py

```diff
@@ -322,9 +331,9 @@ def test_add_route_with_invalid_re(self):
         handler = self.make_handler()
         with self.assertRaises(ValueError) as ctx:
             self.router.add_route('GET', r'/handler/{to:+++}', handler)
-        s = str(ctx.exception)
-        self.assertTrue(s.startswith(
-            "Bad pattern '/handler/(?P<to>+++)': nothing to repeat"), s)
+        self.assertEqual(
+            "Bad pattern '\/handler\/(?P<to>+++)': nothing to repeat",
+            str(ctx.exception))
```

Comment:
Please revert the test: on Python 3.5 error message contains additional info about position in text which causes the error.